odl/set/space.py

```diff
@@ -803,6 +803,16 @@ def __pos__(self):
         """Return ``+self``."""
         return self.copy()
 
+    def __cmp__(self, other):
+        """Comparsion not implemented.
+
+        Raises
+        ------
+        TypeError
+        """
+        raise TypeError('unorderable type `{}`'
+                        ''.format(self.__class__.__name__))
```

Comment:
Can you try to mimic the upstream error message? It says, for example, `unorderable types: int() < str()`. Perhaps replace the comparison operator by a comma, i.e.
```python
raise TypeError('unorderable types: {}, {}'.format(self.__class__.__name__, type(other)))
```